salt/states/boto_lc.py

```diff
@@ -122,6 +122,7 @@ def present(
         kernel_id=None,
         ramdisk_id=None,
         block_device_mappings=None,
+        delete_on_termination=True,
```

Comment:
@nasenbaer13 My concern here is that this results in a behavior change. I know it was documented, but it wasn't implemented as you said. So when folks upgrade to Oxygen, these will start deleting when they didn't before. Perhaps that is still desired, but I wanted to at least bring it up.

What do you think @ryan-lane?

Reply:
So @tkwilliams - I understand you correctly: changing the `delete_on_termination` default to `None` and pass it through should do the trick? At least it shouldn't change the previous behaviour right? But at least it is possible to change it.

Reply:
PR updated, @tkwilliams does it look better now?